rpmbuild/copr-rpmbuild.spec

```diff
@@ -114,21 +118,25 @@ Requires: libsemanage-python
 Requires: nosync
 %ifarch x86_64
 # multilib counterpart to avoid: config_opts['nosync_force'] = True
+%if !0%{?openEuler}
 Requires: nosync(x86-32)
 %endif
+%endif
 Requires: openssh-clients
 Requires: podman
+%if !0%{?openEuler}
 Requires: pyp2rpm
 Requires: pyp2spec
+Requires: rubygem-gem2rpm
+Requires: scl-utils-build
+Requires: fedora-review >= 0.8
+Requires: fedora-review-plugin-java
```

Comment:
Hm, this seems like resolving deployment-specific preferences.  I don't quite like doing this upstream.

Can you please invent something better; that would allow us to specify additional requirements for our deployment (same applies to your instance, how do you keep your deps maintained?).

Reply:
> Hm, this seems like resolving deployment-specific preferences. I don't quite like doing this upstream.
> 
> Can you please invent something better; that would allow us to specify additional requirements for our deployment (same applies to your instance, how do you keep your deps maintained?).

Its a bit ugly for us to ship the ruby/python generator, [we just install the tools from our SCM directly](https://github.com/opensourceways/copr_docker/blob/c79d3e36b5c9633243ac544c395ead8a3442c3c8/docker/builder/Dockerfile#L63-L67) in Dockerfile

Look into the future can we define some virtual provides for ruby/python generator like `system-release`?

And for `fedora-review` and `scl-utils-build`, it should be optional(tools absensed in openEuler), maybe left as `Recommands` so it can be determined at deploy moment?